pkg/types/tuf/v0.0.1/tuf_v0_0_1_schema.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://rekor.sigstore.dev/types/tuf/tuf_v0_0_1_schema.json",
+    "title": "tuf v0.0.1 Schema",
+    "description": "Schema for tuf metadata entries",
+    "type": "object",
+    "properties": {
+        "version": {
+            "description": "TUF specification version",
+            "type": "string",
+            "enum": ["1.0.0"]
+        },
+        "manifest": {
+            "description": "tuf manifest",
```

Comment:
```suggestion
            "description": "TUF manifest",
```

Reply:
You can also fix this for all the generated files by adding the `--additional-initialism=TUF` flag to the `swagger` calls in `Makefile`, like:

```diff
[bcallawa@bcallawa-m900z rekor]$ git diff HEAD Makefile
diff --git a/Makefile b/Makefile
index cb3f873..8718527 100644
--- a/Makefile
+++ b/Makefile
@@ -50,8 +50,8 @@ SERVER_PKG=github.com/sigstore/rekor/cmd/rekor-server/app
 SERVER_LDFLAGS="-X $(SERVER_PKG).gitVersion=$(GIT_VERSION) -X $(SERVER_PKG).gitCommit=$(GIT_HASH) -X $(SERVER_PKG).gitTreeState=$(GIT_TREESTATE) -X $(SERVER_PKG).buildDate=$(BUILD_DATE)"
 
 $(GENSRC): $(SWAGGER) $(OPENAPIDEPS)
-       $(SWAGGER) generate client -f openapi.yaml -q -r COPYRIGHT.txt -t pkg/generated --default-consumes application/json\;q=1
-       $(SWAGGER) generate server -f openapi.yaml -q -r COPYRIGHT.txt -t pkg/generated --exclude-main -A rekor_server --exclude-spec --flag-strategy=pflag --default-produces application/json
+       $(SWAGGER) generate client -f openapi.yaml -q -r COPYRIGHT.txt -t pkg/generated --default-consumes application/json\;q=1 --additional-initialism=TUF
+       $(SWAGGER) generate server -f openapi.yaml -q -r COPYRIGHT.txt -t pkg/generated --exclude-main -A rekor_server --exclude-spec --flag-strategy=pflag --default-produces application/json --additional-initialism=TUF
 ```